### DISCUSSION TITLE
Download seems to be broken for me.

### DISCUSSION THREAD

**Discussion by BadTimeInbound** [GriefPrevention/GriefPrevention#1378] [Q&A]
I can't seem to be able to access the download links at all. It just loads forever on the cloudflare page. Both me and a friend couldn't download anything. Could I get a direct link to the latest version, please?

**Comment by Jikoo** [comment]
DBO appears to be down.

Dev builds can be downloaded off [the CI](https://ci.appveyor.com/project/RoboMWM39862/griefprevention). If you're looking for a specific build, you can click on the green check next to it in the commit history, then click on AppVeyor to pull up the build.
